src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -23,10 +23,10 @@ public class Ado : NotificationsBase, IAdo {
 
         var report = (Report)reportable;
 
-        var notificationInfo = @$"job_id:{report.JobId} task_id:{report.TaskId}
-container:{container} filename:{filename}";
+        var notificationInfo = new Dictionary<string, string>() { { "notification_id", notificationId.ToString() }, { "job_id", report.JobId.ToString() }, { "task_id", report.TaskId.ToString() }, { "ado_project", config.Project }, { "ado_url", config.BaseUrl.ToString() }, { "container", container.String }, { "filename", filename } };
 
-        _logTracer.Info($"notify ado: {report.JobId:Tag:JobId} {report.TaskId:Tag:TaskId} {container:Tag:Container} {filename:Tag:Filename}");
+        var adoEventType = "AdoNotify";
+        _logTracer.Event($"{adoEventType:Tag:AdoEventType} {notificationInfo["notification_id"]:Tag:NotificationId} {notificationInfo["job_id"]:Tag:JobId} {notificationInfo["task_id"]:Tag:TaskId} {notificationInfo["ado_project"]:Tag:AdoProject} {notificationInfo["ado_url"]:Tag:AdoUrl} {notificationInfo["container"]:Tag:Container} {notificationInfo["filename"]:Tag:filename}");
```

Comment:
You could make this serialized string a static function since it seems like it's the same everywhere in this file

Reply:
I updated the notification such that a function no longer makes sense, but it's cleaner!